CITATION.rst

```diff
@@ -3,20 +3,18 @@ Acknowledging or Citing SunPy
 
 If you have used SunPy in your scientific work we would appreciate it if you would acknowledge it.
 The continued growth and development of SunPy is dependent on the community being aware of the use SunPy.
-If you use SunPy, we therefore ask that you acknowledge SunPy appropriately in a publication, presentation, poster, or talk.
 
--  **For a publication**, we recommend the following line be added to
-   the conclusion or acknowledgements:
+Publications
+------------
 
-   *This research has made use of SunPy, an open-source and free
+Please add the following line to the conclusion or acknowledgements:
+
+   *This research has made use of SunPy vX.Y, an open-source and free
    community-developed solar data analysis package written in Python
```

Comment:
Could we shrink this down a little bit with:

`*This research has made use of SunPy vX.Y, an open-source and free community-developed solar data analysis Python package.`?